libs/barista-components/tree-table/src/tree-table-toggle-cell.scss

```diff
@@ -14,10 +14,3 @@
   align-items: center;
   position: relative;
 }
-
```

Comment:
this was not in the changeset of https://github.com/dynatrace-oss/barista/pull/1564/files is this intended?